src/nft/components/sell/modal/ListingSection.tsx

```diff
@@ -0,0 +1,188 @@
+          )}
+          <Column gap="8">
+            {rows.map((row: AssetRow, index) => {
+              return (
+                <Column key={index} gap="8">
+                  <Row>
+                    {row.images.map((image, index) => {
+                      return (
+                        <Box
+                          as="img"
+                          height="20"
+                          width="20"
+                          borderRadius={index === 0 && (row as CollectionRow).collectionAddress ? 'round' : '4'}
+                          style={{ zIndex: 2 - index }}
+                          className={styles.listingModalIcon}
+                          src={image}
+                          alt={row.name}
+                          key={index}
+                        />
+                      )
+                    })}
+                    <Box
+                      marginLeft="8"
+                      marginRight="auto"
+                      fontWeight="normal"
+                      color="blackBlue"
+                      textOverflow="ellipsis"
+                      overflow="hidden"
+                      maxWidth={{
+                        sm: 'max',
+                        md:
+                          row.status === ListingStatus.REJECTED || row.status === ListingStatus.FAILED ? '120' : 'full',
+                      }}
+                      className={bodySmall}
+                    >
+                      {row.name}
+                    </Box>
+                    {isSuccessScreen ? (
+                      getListingRowPrice(row) &&
+                      `${formatEthPrice(numberToWei(getListingRowPrice(row) ?? 0).toString())} ETH`
+                    ) : row.status === ListingStatus.APPROVED ? (
+                      <ApprovedCheckmarkIcon height="20" width="20" />
+                    ) : row.status === ListingStatus.FAILED || row.status === ListingStatus.REJECTED ? (
+                      <Row gap="4">
+                        <Box fontWeight="normal" fontSize="14" color="darkGray">
+                          {row.status}
+                        </Box>
+                        <FailedListingIcon />
+                      </Row>
+                    ) : (
+                      row.status === ListingStatus.SIGNING && <LoadingIcon height="20" width="20" stroke="#4673FA" />
+                    )}
+                  </Row>
+                  {(row.status === ListingStatus.FAILED || row.status === ListingStatus.REJECTED) && (
+                    <Row gap="8" justifyContent="center">
+                      <Box
+                        width="120"
+                        as="button"
+                        className={buttonTextSmall}
+                        borderRadius="12"
+                        border="none"
+                        color="red400"
+                        height="32"
+                        cursor="pointer"
+                        style={{ backgroundColor: '#FA2B391A' }}
```

Comment:
is this supposed to be this color for both light and dark mode?

Reply:
No these are the set colors for try again/failed for both modes and are one off uses